sdk/src/main/scala/com/stratio/sparkta/sdk/Output.scala

```diff
@@ -58,19 +58,19 @@ abstract class Output(keyName: String,
     case Some(fixDimensions) => fixDimensions.split(FieldsSeparator)
   }
 
-  val fixedAgg = properties.getString("fixedAggregation", None)
+  val fixedMeasure = properties.getString("fixedMeasures", None)
```

Comment:
fixedAggregation appears several times in the module. If you do this refactor, rename every reference to this property and update documentation https://stratio.atlassian.net/wiki/display/SPARKTA0x8/